### PR TITLE
Add EOL linter action.

### LINKED ISSUE #142: Line Ending Roundup
Currently, the line endings in this repository are horribly inconsistent and/or broken. For example, flipping the current dataset through plLocalizationEditor reveals many no-op changes. Further, #128 breaks deployment with [Hoikas/UruManifest](https://github.com/Hoikas/UruManifest). Therefore, we need to fix all line endings and add a GHA powered review to automatically request changes for illegal line endings.

### PR DESCRIPTION
**Pull request by Hoikas**
Fixes #142.

### CODE REVIEW

**Comment by Hoikas** [comment]
Hmm... for some reason, the action did not execute on this PR. Any ideas?

**Comment by Hazado** [comment]
No actions setup in moul-assets

**Comment by Hoikas** [comment]
It [worked fine](https://github.com/Hoikas/moul-assets/actions/runs/3718338100) in my fork on push with no actions set up before that.

**Comment by Hazado** [comment]
dont know, maybe because this is a pr and not official part of the repo yet?

im not good with github

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Lint
+on: [push, pull_request]
+
+jobs:
+  check-eol:
+    name: Check Line Endings
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```
Comment (dpogue) [inline .github/workflows/lint.yml:10]: Please use `actions/checkout@v3` to avoid the warning about node12

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+jobs:
+  check-eol:
+    name: Check Line Endings
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
```
Comment (dpogue) [inline .github/workflows/lint.yml:13]: Latest version is `actions/setup-python@v4` also needed to avoid the node12 deprecation warning

**Review by dpogue — APPROVED** [review]

**Pull request merged**